### ISSUE TITLE
Menus don't work in both current dev and release branch

### ISSUE THREAD

**Issue by Strahinja** [skyjake/lagrange#436]
Lagrange compiled from both current dev and current release branches on Artix GNU/Linux doesn't show menus, both context menus when any part of Lagrange's window is right-clicked, and main menu when the main menu "hamburger" button is clicked.

**Comment by skyjake** [comment]
I've heard from two other people also having the same problem. I haven't yet encountered it myself, though.

If you run Lagrange with `-E`, does it say something like this when trying to open the hamburger menu:
```
[command] {1} menu.open ptr:0x55fe4eee4d00
[command] {0} cancel
[command] {1} menu.opened ptr:0x55fe4eee4f10
[command] {1} menu.closed ptr:0x55fe4eee4f10
```

**Comment by skyjake** [comment]
Also, could you attach your CMakeCache.txt?

All reports have been from manual builds, so it may have something to do with the build options.

**Comment by Strahinja** [comment]
I simply followed the installation guide when building Lagrange. That is:
```
$ git clone --recursive --branch release https://git.skyjake.fi/gemini/lagrange
$ cd lagrange
$ mkdir build && cd build
$ cmake .. -DCMAKE_BUILD_TYPE=Release
$ cmake --build .                  # repeating this if necessary
$ sudo cmake --build . --target install
```
Here are the logs. I created _lagrange.log_ by:
```
$ lagrange -E >lagrange.log 2>&1
```
and then tried to open the main menu and context menus a few times.
* [lagrange.log](https://github.com/skyjake/lagrange/files/7874871/lagrange.log)
* [CMakeCache.txt](https://github.com/skyjake/lagrange/files/7874872/CMakeCache.txt)

**Update:** It seems that my right clicks weren't registered above, as my mouse sometimes does that. Here's a log with a bit more attempts:
* [lagrange.log](https://github.com/skyjake/lagrange/files/7874914/lagrange.log)

It also seems that right-clicking the icons above the left panel (bookmarks/favorites/history...) opens the context menu (I opened it once in this attempt), but right-clicking elsewhere (page area, bookmarks in the left panel etc) doesn't.

**Comment by skyjake** [comment]
Thanks!

Do the menus work correctly for you in older releases, like the git tags `v1.8.3` or `v1.7.3`?

**Comment by Strahinja** [comment]
Just tried
```
$ git checkout v1.8.3
```
built it, and it has the same issue as the current version. Same with v1.7.3.

**Comment by skyjake** [comment]
Please test with the latest `dev` branch.

**Comment by Strahinja** [comment]
I tested the latest dev branch, and the menus work, but they are somewhat oddly placed. Example with main menu, which covers the toolbar (even when it has space to fully show the menu below the toolbar); context menu feels a bit off too, but maybe that is subjective:
![2022-01-16-141219-dwm](https://user-images.githubusercontent.com/981077/149661503-a8cf1753-7fc9-4082-84ac-f8f1f399b599.png)

**Update:** After checking how exactly context menu placement works in other programs, I think that the context menu placement is ok. Only the placement of main menu could be adjusted to be below toolbar.

**Comment by skyjake** [comment]
I have now placed the main menu under the hamburger button, where one would expect it to be.

This was actually its original place, but I moved it up over the button as a compromise when dealing with some other minor issues.

**Comment by skyjake** [comment]
The fix for menus not opening will be included in the upcoming v1.10 (to be released soon). Backporting to earlier stable releases is possible if needed.

**Comment by Strahinja** [comment]
Thank you!
